src/logger/loggerApi.c

```diff
@@ -1352,6 +1347,133 @@ int api_setCanConfig(struct Serial *serial, const jsmntok_t *json)
     return API_SUCCESS;
 }
 
+static void _json_can_mapping(struct Serial *serial, CANMapping *mapping, int more)
+{
+        json_bool(serial, "bm", mapping->bit_mode, 1);
+        json_int(serial, "bus", mapping->can_channel, 1);
+        json_int(serial, "id", mapping->can_id, 1);
+        json_int(serial, "id_mask", mapping->can_mask, 1);
+        json_bool(serial, "bigEndian", mapping->big_endian, 1);
+        json_int(serial, "offset", mapping->offset, 1);
+        json_int(serial, "len", mapping->length, 1);
+        json_float(serial, "mult", mapping->multiplier, DEFAULT_CAN_MAPPING_PRECISION, 1);
+        json_float(serial, "div", mapping->divider, DEFAULT_CAN_MAPPING_PRECISION, 1);
+        json_float(serial, "add", mapping->adder, DEFAULT_CAN_MAPPING_PRECISION, 1);
+        json_int(serial, "type", mapping->type, 1);
+        json_int(serial, "filtId", mapping->conversion_filter_id, more);
+}
+
+int api_get_can_channel_config(struct Serial *serial, const jsmntok_t *json)
+{
+    CANChannelConfig * can_channel_cfg = &(getWorkingLoggerConfig()->can_channel_cfg);
```

Comment:
:+1:   I also realized I could use more const as for these new function parameters, too.